### PR TITLE
DEVDOCS-5655: [update] b2b-company.yml

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-5655]

## What changed?
Updated company.yml

## Release notes draft
N/A
## Anything else?

ping {names}

[DEVDOCS-5655]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5655?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/b2b-edition/specs/storefront/storefront/company.yaml
Comment (bc-micah) [inline docs/b2b-edition/specs/storefront/storefront/company.yaml:2810]: @bc-traciporter - These items marked as `x-internal` are supposed to be only visible to internal team as they are used by the b2b edition control panel project and not used externally.

File: docs/b2b-edition/specs/storefront/storefront/company.yaml
Comment (bc-micah) [inline docs/b2b-edition/specs/storefront/storefront/company.yaml:786]:
```suggestion
      operationId: get-companies-validations-storefront-user-email
```

looks like this has typos, and instead of `frontend` we should talk about this as a `storefront` operation

File: docs/b2b-edition/specs/storefront/storefront/company.yaml
Comment (bc-micah) [inline docs/b2b-edition/specs/storefront/storefront/company.yaml:788]: we should reference this as `storefront` instead of `front-end`

**Review by bc-micah — CHANGES_REQUESTED** [review]

**Comment by bc-micah** [comment]
Please help review this - @bigcommerce/team-b2b

**Comment by bc-traciporter** [comment]
Hi @bc-micah I wonder if there is a mistake for the GET /users/validations/existence endpoint. There is a request body for this GET endpoint and no response body. This seems opposite. See lines 4503 and 4533.

File: docs/b2b-edition/specs/storefront/storefront/company.yaml
Comment (bc-traciporter) [reply docs/b2b-edition/specs/storefront/storefront/company.yaml:786]: @bc-micah The endpoint shows  "/companies/validations/fronted/user-emails/{email}"
Is this correct? So I would need to change the URL, summary on line 697, operationId and description from front-end to storefront? Please clarify if the endpoint URL will change.

Comment (becomevocal) [reply docs/b2b-edition/specs/storefront/storefront/company.yaml:778]: @bc-micah looking at the B2B API codebase, it looks like the endpoint in the swagger file is accurate (it's misspelled) so we'll leave it like this for now. If you create a new alias for the endpoint that is storefront, we can update!

**Review by becomevocal — DISMISSED** [review]

File: docs/b2b-edition/specs/storefront/storefront/company.yaml
Comment (becomevocal) [inline docs/b2b-edition/specs/storefront/storefront/company.yaml:15]:
```suggestion
        Create a company on the storefront.
```

File: docs/b2b-edition/specs/storefront/storefront/company.yaml
Comment (becomevocal) [reply docs/b2b-edition/specs/storefront/storefront/company.yaml:780]: Simplifying this so it doesn't reference front-end. After all, the responses should be in the definition of the response, not the description.

```suggestion
        Check if the user email is valid for current company.
```

**Comment by becomevocal** [comment]
@bc-micah can you re-review this? We can't merge without an approval since you requested changes 🙇

**Review by becomevocal — APPROVED** [review]

**Review by bc-micah — APPROVED** [review]

**Pull request merged**
